set.mm

```diff
@@ -326679,10 +326679,11 @@ vector spaces which are also normed vector spaces (that is, normed groups
     ipcau.v $e |- V = ( Base ` W ) $.
     ipcau.h $e |- ., = ( .i ` W ) $.
     ipcau.n $e |- N = ( norm ` W ) $.
-    $( The Cauchy-Schwarz inequality for a subcomplex pre-Hilbert space.
-       (Contributed by Mario Carneiro, 11-Oct-2015.) $)
+    $( The Cauchy-Schwarz inequality for a subcomplex pre-Hilbert space.  This
+       is Metamath 100 proof #78.  (Contributed by NM, 12-Jan-2008.)  (Revised
+       by Mario Carneiro, 11-Oct-2015.) $)
```

Comment:
Maybe the note about naming (*Schwarz inequality*, *Bunjakovaskij-Cauchy-Schwarz inequality*) should go here, rather than into the MM#100 page?

It would also be nice to include the reference: `Part of Lemma 3-2.1(a) of [Kreyszig] p. 137.`, if it is valid.

Reply:
Alternative names: probably more than half of the 100 theorems have at least two alternative names, so why single out this one. We are not a "history of math" website, and I think that kind of notes may even leave an amateurish impression on readers.

Book reference: I think it's not needed for such a standard result.  If however that book is really followed in large parts of the database, I can add it back. Do you know ?

Reply:
> Alternative names: probably more than half of the 100 theorems have at least two alternative names, so why single out this one. 
 
Indeed, Ok.

> Book reference: I think it's not needed for such a standard result. If however that book is really followed in large parts of the database, I can add it back. Do you know ?

`[Kreyszig]` Appears 75 times in the database, and only the last 10 or so are in the obsolete "new usage discouraged" section, so I'd say it's followed pretty well.

Reply:
(I use "rocket" for "done".)